### DISCUSSION TITLE
Ran the x86_64 version but no audio

### DISCUSSION THREAD

**Discussion by guilherme-leite** [FundamentalFrequency/.github#52] [Q&A]
Hey, I'm pretty new to all this. 

Tried to download the x86_64 emulator on ubuntu and and it ran but with no audio. Any clues of what I'm doing wrong? 
Do I need to set up anything else in order to use the emulator version?

Thanks

**Comment by stonepreston** [comment]
The emulator only emulates the MIDI functionality of the LMN-3, it does not output any audio. It only spits out MIDI data. You need to also run the x86-64 version of the [LMN-3 DAW](https://github.com/FundamentalFrequency/LMN-3-DAW) at the same time as the emulator (and on the same machine). Then you can use the emulator to control the DAW.

**Reply by guilherme-leite** [reply]
Thank you for the help! Will get right on it!
